boards/same54-xpro/include/board.h

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (C) 2017 Travis Griggs <travisgriggs@gmail.com>
```

Comment:
Isn't this a new file? If yes, then why not write a new copyright line? And then you don't need to copy authors (see below).